skt/runner.py

```diff
@@ -227,6 +227,46 @@ def getverboseresults(self, joblist):
 
         return result
 
+    def get_ltp_lite_logs(self, job_id):
+        """
+        Get logs produces by LTP lite, for each recipe in specified job.
+
+        Args:
+            job_id: ID of the job for which to retrieve the logs.
+
+        Returns:
+            A dictionary in format
+              R:<recipe_id>: (<link_to_full_run_log>, <link_to_results>),
+            for each recipe in the job.
+        """
+        job_results = self.getresultstree(job_id, True)
+        ltp_logs = {}
+
+        for recipe in job_results.findall('recipeSet/recipe'):
+            recipe_id = 'R:%s' % recipe.attrib.get('id')
+
+            ltp_node = recipe.find(
+                "task[@name='/kernel/distribution/ltp/lite']"
+            )
+            if ltp_node is None:
```

Comment:
`xpath` could help a lot here to avoid lots of `if` statements, but this is okay for now. Dealing with XML is a huge pain.